lib/auth/permissions.go

```diff
@@ -234,6 +234,7 @@ func (a *authorizer) authorizeRemoteBuiltinRole(r RemoteBuiltinRole) (*Context,
 		services.RoleSpecV3{
 			Allow: services.RoleConditions{
 				Namespaces: []string{services.Wildcard},
+				AppLabels:  services.Labels{services.Wildcard: []string{services.Wildcard}},
```

Comment:
This fixes TestAppAccessForward, does this permission seem reasonable?

Reply:
Unclear how this would have any impact, because the `Rules` section does not list KindApp

Reply:
> Unclear how this would have any impact, because the `Rules` section does not list KindApp

The default implicit role has a rule for RO access to KindAppServer, which allows this RoleSet to pass the actions check but potentially fail the labels check